ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -345,7 +345,6 @@ export interface WorkspaceEditState {
   loading: boolean;
   showUnderservedPopulationDetails: boolean;
   showStigmatizationDetails: boolean;
-  reviewRequestedHasSelection: boolean;  // false initially, set to true once the user has chosen (either Yes or No)
```

Comment:
Now it is pure binary: either Yes or No will always be selected.  This matches the persisted state of the Workspace.

Reply:
+1
I don't even know what my opinion is on nullable booleans.